apps/spruce/src/pages/projectSettings/Tabs.tsx

```diff
@@ -272,8 +272,11 @@ export const ProjectSettingsTabs: React.FC<Props> = ({
           element={
             <AppSettingsTab
               githubPermissionGroups={
-                projectData?.projectRef?.githubDynamicTokenPermissionGroups ??
-                []
+                projectType === ProjectType.Repo
+                  ? (repoData?.projectRef?.githubDynamicTokenPermissionGroups ??
+                    [])
+                  : (projectData?.projectRef
+                      ?.githubDynamicTokenPermissionGroups ?? [])
```

Comment:
i think it makes sense to merge them
```suggestion
                [
                ...(projectData?.projectRef
                  ?.githubDynamicTokenPermissionGroups ?? []),
                ...(repoData?.projectRef?.githubDynamicTokenPermissionGroups ??
                  []),
              ]
```